how-to-guides/re-run-failed-training/notebooks/re_run_failed_training.ipynb

```diff
@@ -0,0 +1,783 @@
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-WbPjLDk1GcC"
+      },
+      "source": [
+        "## Step 3: Fetching and downloading data from Neptune\n",
+        "\n",
+        "Fetch metadata(dataset and hyperparameters) needed to re-run the training. Precisely, you will download the hyperparameters used in the failed run to instantiate a model with the same configuration and then you will download the dataset path to get the same dataset too.\n",
+        "\n",
+        "To do that:"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_E_T5gUvA6kP"
+      },
+      "source": [
+        "### Use the  [.download()](https://docs.neptune.ai/api-reference/field-types#.download-3) method to retrieve the [dataset artifact](https://docs.neptune.ai/api-reference/field-types#artifact) to your local disk:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "XcfHQqomSrAT"
+      },
+      "outputs": [],
+      "source": [
+        "data_dir = 'data'"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 21,
+      "metadata": {
+        "id": "8j0cP5ZTA5AJ"
+      },
+      "outputs": [],
+      "source": [
+        "failed_run['artifacts/dataset'].download(destination=data_dir)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BVJKNQxIYPAC"
+      },
+      "source": [
+        "### Use the [.fetch()](https://docs.neptune.ai/api-reference/field-types#.fetch-1) method to retrieve hyperparameters:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 14,
+      "metadata": {
+        "id": "k_8vefGCNBIJ"
+      },
+      "outputs": [],
+      "source": [
+        "# fetching non-file values \n",
+        "failed_run_params = failed_run['config/hyperparameters'].fetch()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "RxC8thpFHl-H"
+      },
+      "source": [
+        "## Step 4: Create a new run\n",
+        "Create a new Neptune run that will be used to log metadata in the re-run session."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 17,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "ruRMeJUiHWE7",
+        "outputId": "b2f38fed-0482-4063-c426-657d92ea907e"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "https://app.neptune.ai/common/showroom/e/SHOW-3296\n",
+            "Remember to stop your run once you’ve finished logging your metadata (https://docs.neptune.ai/api-reference/run#stop). It will be stopped automatically only when the notebook kernel/interactive console is terminated.\n"
+          ]
+        }
+      ],
+      "source": [
+        "new_run = neptune.init(\n",
```

Comment:
Please, keep order of arguments as they appear in the docs -> standardization.
https://docs.neptune.ai/api-reference/neptune#.init_run